stix_shifter_modules/gcp_chronicle/stix_transmission/api_client.py

```diff
@@ -0,0 +1,160 @@
+from google.oauth2 import service_account
+from googleapiclient import _auth
+from google.auth.exceptions import RefreshError
+from httplib2 import ServerNotFoundError
+import json
+from stix_shifter_utils.utils import logger
+from stix_shifter_utils.utils.error_response import ErrorResponder
+
+
+class InvalidResponseException(Exception):
+    pass
+
+
+class APIClient:
+    QUERY_ENDPOINT = 'v2/detect/rules'
+    QUERY_SCOPES = ['https://www.googleapis.com/auth/chronicle-backstory']
+    URI = "https://oauth2.googleapis.com/token"
+
+    def __init__(self, connection, configuration):
+        self.auth = configuration.get('auth')
+        self.auth['private_key'] = connection.get('selfSignedCert').replace('\\n', '\n')
```

Comment:
The original private_key comes with \n. It can be either in single line with \n or it can come with newline also. In order to handle private_key with or without \n, this replace statement is required.